# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bin86/no-elks.patch.cross
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dev86-0.16.17/Makefile.vanilla	2007-04-10 09:12:53.000000000 +0200
+++ dev86-0.16.17/Makefile	2007-04-10 09:13:01.000000000 +0200
@@ -5,11 +5,11 @@
 VERSION=0.16.17
 
 TARGETS=install clean other \
-    bcc86 unproto copt as86 ld86 elksemu \
+    bcc86 unproto copt as86 ld86 \
     install-all install-bcc install-emu install-lib \
     install-lib2 install-ln install-man install-other \
     all-libs alt-libs library lib-386 lib-bsd lib-dos lib-fast lib-stand \
-    config tests dis88 doselks bootblocks ld86r
+    config tests dis88 bootblocks ld86r
 
 ELKSSRC= /usr/src/elks
 PREFIX=  /usr
--- dev86-0.16.17/makefile.in.vanilla	2007-04-10 09:33:27.000000000 +0200
+++ dev86-0.16.17/makefile.in	2007-04-10 09:33:34.000000000 +0200
@@ -89,10 +89,10 @@
 
 #ifdef GNUMAKE
 all: check_config bcc86 cpp unproto copt as86 ar86 ld86 objdump86 \
-     library lib-bsd alt-libs elksemu
+     library lib-bsd alt-libs
 
 install: check_config install-bcc install-man \
-     install-lib install-emu
+     install-lib
 
 install-all: install install-other
 
